src/client/services/Network.ts

```diff
@@ -0,0 +1,77 @@
+import { Client, Room } from 'colyseus.js'
+import Phaser from 'phaser'
+import { IOfficeState, IPlayer } from '../../types/IOfficeState'
+import { Message } from '../../types/Messages'
+
+export default class Network {
+  private client: Client
+  private room?: Room<IOfficeState>
+  private events: Phaser.Events.EventEmitter
+
+  private _mySessionId!: string
+  set mySessionId(id: string) {
+    this._mySessionId = id
+  }
+  get mySessionId() {
+    return this._mySessionId
+  }
+
+  constructor() {
+    const protocol = window.location.protocol.replace('http', 'ws')
+    const endpoint =
+      process.env.NODE_ENV === 'production'
+        ? `wss://sky-office.herokuapp.com`
+        : `${protocol}//${window.location.hostname}:2567`
+    this.client = new Client(endpoint)
+    this.events = new Phaser.Events.EventEmitter()
```

Comment:
This initialization does not depend on anything in the constructor, move it to initialize inline:

```ts
  private events = new Phaser.Events.EventEmitter()
```

Reply:
ok!